addons/stock/migrations/13.0.1.1/post-migration.py

```diff
@@ -185,6 +185,63 @@ def map_stock_location_usage(env):
     )
 
 
+def map_stock_picking_responsible_responsible_id_to_user_id(env):
+    """
+    responsible_id (partner_id) field in stock_picking_responsible is replaced by user_id (res.users)
+    We create a deactivated user for partners without user and then
+    map the partner to their user in the stock picking.
+    """
+
```

Comment:
**nitpicking:** Don't put empty lines inside a method.

Reply:
I assumed you mean only the first empty line, the others give some air and enhance readability.

Reply:
I mean all, as syntax highlighting already makes the needed visual separations. It's also recommended since PEP8 to not do it (although tolerated), and conceptually, if you need to have a code block, put a comment before that acts as separator and explaining what the block does. Anyway, it's not blocking by my part.

Reply:
Ok, I'll leave it as is then :-)